src/osuTK/Input/JoystickState.cs

```diff
@@ -72,7 +73,10 @@ public float GetAxis(int axis)
         /// <param name="button">The button to query.</param>
         public ButtonState GetButton(int button)
         {
-            return (buttons & ((long)1 << button)) != 0 ? ButtonState.Pressed : ButtonState.Released;
+            unsafe
+            {
+                return buttons[button] ? ButtonState.Pressed : ButtonState.Released;
```

Comment:
should be guarded against out-of-bounds access (if a joystick with >128 buttons is presented). probably just returning false.